tripy/examples/segment-anything-model-v2/README.md

```diff
@@ -6,23 +6,24 @@ This is an implementation of SAM2 model ([original repository](https://github.co
 
 ## Running The Example
 
-### Image pipeline
-
 1. Install prerequisites:
 
     ```bash
-    sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6  -y
+    sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6 jpeginfo -y
     python3 -m pip install -r requirements.txt
     ```
 
-2. Retrieve an example image and the checkpoint:
+2. Retrieve the images and the checkpoint:
 
     ```bash
-    wget -O truck.jpg https://raw.githubusercontent.com/facebookresearch/sam2/main/notebooks/images/truck.jpg
+    chmod +x download_images.sh
```

Comment:
Let's just ship it with the executable bit set?

Reply:
Or maybe even make it a Python script so it's easier to support this on Windows in the future